api/db/db.go

```diff
@@ -0,0 +1,35 @@
+package db
+
+import (
+	"log"
+	"os"
+
+	pg "gopkg.in/pg.v5"
+)
+
+func NewDB() *pg.DB {
+	user := os.Getenv("DATABASE_USER")
```

Comment:
This will eventually need to read from VCAP_SERVICES in prod and staging (and from env vars in local/test).